_data/chains/eip155-634.json

```
@@ -0,0 +1,24 @@
+{
+  "name": "Avocado",
+  "chain": "Avocado",
+  "rpc": ["https://rpc.avocado.instadapp.io"],
+  "faucets": [],
+  "nativeCurrency": {
+    "name": "USDC",
+    "symbol": "USDC",
+    "decimals": 18
+  },
+  "infoURL": "https://avocado.instadapp.io",
+  "shortName": "avocado",
+  "chainId": 634,
+  "networkId": 634,
+  "icon": "avocado",
+  "explorers": [
+    {
+      "name": "avoscan",
+      "url": "https://avoscan.co/",
+      "icon": "avocado",
+      "standard": "EIP3091"
```

explorer does not seem to be EIP3091 compatible 
```suggestion
      "standard": "none"
```